src/BlazorUI/Bit.BlazorUI/Components/Lists/BasicList/BitBasicList.scss

```
@@ -1,12 +1,81 @@
-﻿@import "../../../Styles/functions.scss";
+@import "../../../Styles/functions.scss";
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)